e2e/global-setup-node-modules/setup.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {compact} from 'lodash-es';
```

Comment:
eslint doesn't seem to like this import. Not sure how to best solve it... Maybe add `lodash-es` to the root `package.json` so it's present before the tests run?